packages/block/test/block.spec.ts

```diff
@@ -20,7 +20,7 @@ import util from 'util'
 
 tape('[Block]: block functions', function (t) {
   t.test('should test block initialization', function (st) {
-    const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Chainstart })
+    const common = new Common({ chain: ChainId.Mainnet, hardfork: HardforkName.Chainstart })
```

Comment:
Ah, sorry, but I find both these renaming very unfortunate, I didn't think that you would change these actually.

These are *all* over the place, we settled on the short names a long time ago (and I honestly also like these a lot) and this is also used by the Community on all their instantiations, so this would cause a lot of unnecessary hazzle.

Can you please name back? If this conflicts with something else, then please rename the "else" part, but this should really stay stable and untouched.

Reply:
Yes, these are a lot more uncritical, since close to no one will use these directly. So maybe we can do `HardforkConfig` and `ChainConfig` for these?

Reply:
(not sure if we should then rename to `GenesisBlockConfig` and `BootstrapNodeConfig` as well for consistency? A bit undecided)

Reply:
I believe I've addressed all the feedback here.  I know @ryanio mentioned combining the types/enums into one file but I kind of like them separated.  Just makes things more readable to my way of thinking but could be persuaded otherwise.

Reply:
nice thanks, just took a look, yeah i think it is kind of nice to have them separated actually :)